worker/src/RTC/RtpStreamRecv.cpp

```diff
@@ -611,12 +611,11 @@ namespace RTC
 		if (this->rtt > 0.0f)
 		{
 			this->hasRtt = true;
-		}
-
-		// Tell it to the NackGenerator.
-		if (this->params.useNack)
-		{
-			this->nackGenerator->UpdateRtt(static_cast<uint32_t>(this->rtt));
+			// Tell it to the NackGenerator.
+			if (this->params.useNack)
+			{
+				this->nackGenerator->UpdateRtt(static_cast<uint32_t>(this->rtt));
```

Comment:
Why are you just updating current RTT value in `NackGenerator` **only** if > 0.0f? This is wrong. It means that if negative then `NackGenerator` will keep using its previous given RTT value.